source/core/src/main/com/csse3200/game/components/player/PlayerInventoryDisplay.java

```diff
@@ -34,9 +34,11 @@ public class PlayerInventoryDisplay extends UIComponent {
     private boolean toggle = false; // Whether inventory is toggled on;
     DialogueBox itemOverlay;
 
+    private boolean toggleHotbar = true;
 
-    private final Skin skininv=new Skin(Gdx.files.internal("Inventory/inventory.json"));
-    private final Skin skinSlots=new Skin(Gdx.files.internal("Inventory/skinforslot.json"));
+
+    private final Skin skininv=new Skin(Gdx.files.internal("Inventory/inventory.json"));//created by @PratulW5
+    private final Skin skinSlots=new Skin(Gdx.files.internal("Inventory/skinforslot.json"));//created by @PratulW5
 
```

Comment:
Adding slot listeners should not be a void function (ie anyone can access it). At least make it public void